back-end/services/AuthService.js

```diff
@@ -2,8 +2,8 @@ const User = require('../models/User');
 const bcrypt = require('bcryptjs');
 const jwt = require('jsonwebtoken');
 
-const register = (data) => {
-  bcrypt.hash(data.password, 10, (err, hashedPass) => {
+const register = async (data) => {
+  await hashPasswords(data.password, 10, (err, hashedPass) => {
```

Comment:
Are you sure that this function will be work? Function `hashPasswords` needed 1 parameter, but you pass 3.

Reply:
it works properly, tried it with registration